firmware/motorcontroller2/code/app/messageHandler_componentSpecific.h

```diff
@@ -8,21 +8,24 @@
 #ifndef MESSAGEHANDLER_COMPONENTSPECIFIC_H_
 #define MESSAGEHANDLER_COMPONENTSPECIFIC_H_
 
+#define USE_CAN     (0U)
+#define USE_UART    (1U)
+
 typedef enum
 {
-    MESSAGE_HANDLER_RX_MESSAGE_CHANNEL_POLARIS_REQUEST,
-    MESSAGE_HANDLER_RX_MESSAGE_CHANNEL_MOTOR_SPEED,
+    MESSAGE_HANDLER_RX_CHANNEL_POLARIS_REQUEST,
```

Comment:
Change to trident.

Reply:
change all instances of polaris to trident. Polaris is retired.

Reply:
Will do. Didn't you change this in the protocol.h file? I'm still seeing POLARIS there.